docs/source/tutorial/create-schema.rst

```diff
@@ -0,0 +1,168 @@
+Example: Create a Schema for Dakota's Height Journey
+----------------------------------------------------
+
+Dakota is 17 years old. She has measured her heights every year since she was 8. She decides to contribute her height
+history for scientific research. She recorded her height history in a CSV table:
+::
+
+   age,height
+   8,130
+   9,135
+   10,141
+   11,142
+   12,150
+   13,159
+   14,164
+   15,172
+   16,174
+   17,175
+
+She wrote a README document: ::
+
+   Dakota's height history. Heights are in centimeters.
+
+   To the extent possible under law, Dakota have dedicated all copyright and related and neighboring rights to this
+   dataset worldwide. You should have received a copy of the CC0 Public Domain Dedication along with this software. If
+   not, see <http://creativecommons.org/publicdomain/zero/1.0/>.
+
+She then packed the data file and the README to a tarball ``dakota-height-history.tar.gz``:
+
+.. code-block:: console
+
+   $ tar tf dakota-height-history.tar.gz
+   README
+   height.csv
+
+She then uploaded the tarball to https://pydax.readthedocs.io/en/latest/_static/dakota-height-history.tar.gz. To ease
```

Comment:
Can we replace the URL with a hyperlinked text instead? having a full URL in the main page might look a bit cluttery.